### PR TITLE
Add initial commit for addint support for kubernetes node

### PR DESCRIPTION
**Pull request by mabuaisha**

### CODE REVIEW

**Review by EarthmanT — APPROVED** [review]

**Review by EarthmanT — APPROVED** [review]

**Review by 0lvin — APPROVED** [review]

File: aws.yaml [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+  This blueprint creates a Kubernetes Cluster.
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/4.3/types.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-diamond-plugin/1.3.8/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-fabric-plugin/1.5.1/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-incubator/cloudify-utilities-plugin/1.5.2/plugin.yaml
+  - imports/kubernetes.yaml
+  - https://github.com/cloudify-cosmo/cloudify-aws-plugin/releases/download/1.5.1.2/plugin.yaml
+  - imports/cloud-config.yaml
```
Comment (EarthmanT) [inline aws.yaml]: Use 4.3 plugin import syntax, e.g.:
```
  - plugin:cloudify-diamond-plugin
  - plugin:cloudify-fabric-plugin
  - plugin:cloudify-utilities-plugin
  - plugin:cloudify-kubernetes-plugin
  - plugin:cloudify-awssdk-plugin
```

**Review by 0lvin — CHANGES_REQUESTED** [review]
Remove any binary files, and put to "release attachment to provider."

**Comment by mabuaisha** [comment]
@0lvin Yeah this should be removed I'm adding this for testing and will remove it once approve everything

**Review by 0lvin — APPROVED** [review]
Looks good. CR done.

**Pull request merged**